flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/utils/SourceRecordUtils.java

```diff
@@ -44,10 +44,16 @@ public class SourceRecordUtils {
 
     private SourceRecordUtils() {}
 
-    public static final String SCHEMA_CHANGE_EVENT_KEY_NAME =
+    public static final String MYSQL_SCHEMA_CHANGE_EVENT_KEY_NAME =
```

Comment:
Could you help to refactor this code ? The information about mysql should not be found in base module.

Reply:
I try it.